src/gpodder/gtkui/main.py

```diff
@@ -678,14 +678,17 @@ def on_find_podcast_activate(self, *args):
 
     def init_podcast_list_treeview(self):
         size = cake_size_from_widget(self.treeChannels) * 2
-        self.podcast_list_model.set_max_image_size(size)
+        scale = self.treeChannels.get_scale_factor()
```

Comment:
Apparently this API is only in GTK 3.10 and above, not sure which GTK version is the minimum supported version for gPodder. If needed I can test if this is available. Suggestions welcome :smiley:

Reply:
looks good to me.
Indeed min gtk+ version is not tracked (macOS bundle is 3.24). Let's add a Gtk+ requirement line in the readme (after pygobject).

Reply:
Done